pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -51,6 +55,30 @@ message ClientKeepaliveConfiguration {
   bool permit_without_stream = 3;
 }
 
+message ClientOAuthConfiguration {
+  // Use default google credentials (false by default)
```

Comment:
I suspect it would make more sense to model `google_default_credentials` and `json_key` as a `oneof`:
``` proto
oneof credentials {
  google.protobuf.Empty google_default_credentials = 1;
  string json_key = 2; // Or maybe call this 'service_account' to match the terminology of the OAuth library?
}
```
This ensures that they are truly orthogonal.

Reply:
will add

Reply:
added in the next commit